src/solace_agent_mesh/gateway/http_sse/alembic/versions/20251008_safe_projects_migration.py

```diff
@@ -28,12 +28,10 @@ def upgrade() -> None:
         op.create_table('projects',
             sa.Column('id', sa.String(), nullable=False),
             sa.Column('name', sa.String(), nullable=False),
-            sa.Column('user_id', sa.String(), nullable=True),
```

Comment:
I think we can simplify created_by_user_id and user_id to be the same value since we are removing the notion of templates